tests/pages/faretype.test.tsx

```diff
@@ -1,13 +1,12 @@
-/* eslint-disable global-require */
-
 import * as React from 'react';
 import { shallow } from 'enzyme';
-import FareType from '../../src/pages/faretype';
+// eslint-disable-next-line import/no-unresolved
```

Comment:
We should remove this `eslint-disable` because it is a valid complaint by the linter - the filenames which reference 'faretype' haven't changed to the new 'fareType'

Reply:
Yep good shout.

Reply:
Needed to leave it in to be able to commit, though.